notebooks/00_quick_start/sar_aml_compute_movielens.ipynb

```diff
@@ -0,0 +1,477 @@
+    "                      col_rating=\"Rating\", col_prediction=\"prediction\", \n",
+    "                      relevancy_method=\"top_k\", k=args.top_k)\n",
+    "eval_precision = precision_at_k(test, top_k, col_user=\"UserId\", col_item=\"MovieId\", \n",
+    "                                col_rating=\"Rating\", col_prediction=\"prediction\", \n",
+    "                                relevancy_method=\"top_k\", k=args.top_k)\n",
+    "eval_recall = recall_at_k(test, top_k, col_user=\"UserId\", col_item=\"MovieId\", \n",
+    "                          col_rating=\"Rating\", col_prediction=\"prediction\", \n",
+    "                          relevancy_method=\"top_k\", k=args.top_k)\n",
+    "\n",
+    "run.log(name=\"Model\", value=model.model_str)\n",
+    "run.log(name=\"Top K\", value=args.top_k)\n",
+    "run.log(name=\"MAP\", value=eval_map)\n",
+    "run.log(name=\"NDCG\", value=eval_ndcg)\n",
+    "run.log(name=\"Precision@K\", value=eval_precision)\n",
+    "run.log(name=\"Recall@K\", value=eval_recall)\n",
+    "\n",
+    "pm.record(\"map\", eval_map)\n",
+    "pm.record(\"ndcg\", eval_ndcg)\n",
+    "pm.record(\"precision\", eval_precision)\n",
+    "pm.record(\"recall\", eval_recall)\n",
+    "pm.record(\"train_time\", train_time)\n",
+    "pm.record(\"test_time\", test_time)\n",
+    "\n",
+    "os.makedirs('outputs', exist_ok=True)\n",
+    "# note file saved in the outputs folder is automatically uploaded into experiment record\n",
+    "joblib.dump(value=model, filename='outputs/movielens_sar_model.pkl')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'./movielens-sar/reco_utils/'"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import shutil\n",
+    "shutil.rmtree('./movielens-sar/reco_utils/', ignore_errors=True)\n",
+    "shutil.copytree('../../reco_utils/', './movielens-sar/reco_utils/')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create an estimator\n",
+    "An estimator object is used to submit the run.  Create your estimator by running the following code to define:\n",
+    "\n",
+    "* The name of the estimator object, `est`\n",
+    "* The directory that contains your scripts. All the files in this directory are uploaded into the cluster nodes for execution. \n",
+    "* The compute target.  In this case you will use the AmlCompute you created\n",
+    "* The training script name, train.py\n",
+    "* Parameters required from the training script \n",
+    "* Python packages needed for training\n",
+    "\n",
+    "In this tutorial, this target is AmlCompute. All files in the script folder are uploaded into the cluster nodes for execution. The data_folder is set to use the datastore (`ds.as_mount()`)."
```

Comment:
Why are you using as_mount() vs as_download()? We should put more details to explain why we are doing certain things in AzureML

Reply:
Added more explanation with a link to datastore documentation